book/website/community-handbook/contributing.md

```diff
@@ -108,33 +104,65 @@ You can also read the [Authoring section of the MyST Markdown Guide](xref:myst-g
 (ch-contributing-pull-request)=
 ## Making Changes with a Pull Request
 
-Changes to the book are always made through pull requests.
+Changes to the book are always made through {term}`pull requests <pull request>`.
 We encourage all contributors to work on their own fork of the repository.
 Pull requests should be self-contained and as small as possible.
 
-Pull request should be marked as drafts until there are ready for review.
+Pull request should be marked as drafts until there are ready for {term}`review`.
 
 We follow the [GitHub flow](https://docs.github.com/en/get-started/using-github/github-flow) workflow:
 
 1. Create a feature branch on your fork
 2. Commit changes
 3. Open a pull request to _The Turing Way_'s repository
   - Open a draft pull request if the work is still in progress
-  - Open a pull request if you feel the work is ready for review
+  - Open a pull request if you feel the work is ready for {term}`review`
 4. Iterate on changes in response to CI and reviewer comments
```

Comment:
Just caught that "CI" is jargon and probably needs to be added to the glossary 😩